getstarted.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default-nav
+title: Get Started
+description: mlpack | fast, flexible machine learning library in C++
+order: 2
+---
+
+<div class="page-title-header">Get Started</div>
+
+<h1>Overview</h1>
+
+There are multiple ways to get mlpack up and running. Python bindings can be installed using pip or conda, or built from source (see "Build from source" tutorials).
+For C++, if mlpack is not available via your preferred OS package manager, or if you need to build your own version (e.g. to apply optimizations, use a different 
+set of BLAS/LAPACK, or build a different configuration), please refer to the "Build from source" tutorials.
+For Windows, prebuilt binaries will help you get started without the need of building mlpack. These packages include both the C++ mlpack library as well as the CLI tools.
+
+Here is a summary of the currently available distribution options you can use depending on your needs:
+
+<h2>Python</h2>
+
+<h3> Get via pip or conda </h3>
+- <a href="doc/mlpack-3.1.0/doxygen/python_quickstart.html"> mlpack in Python quickstart guide </a>
```

Comment:
Agree, but is that true for pip? I tried that last week and didn't work for me.

Also, regarding the package manager list, that was the original idea that @zoq started to work with, but I guess it requires additional effort to first ensure that packages are published/updated for each distribution. I don't own that master list, but it would be nice to start compiling it.

Again, as soon as we have that, adding it to the website is trivial!

Reply:
On Windows I haven't managed to make pip work yet for mlpack, but it would work for Linux and OS X.  I agree, once we get the list together adding it would be easy for sure. :+1: